kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -94,8 +94,9 @@ import           Kore.Internal.TermLike
                  ( TermLike )
 import qualified Kore.Internal.TermLike as TermLike
 import qualified Kore.Logger as Logger
-import           Kore.OnePath.Verification
-                 ( Claim, CommonProofState )
+import           Kore.ProofState
+                 ( ProofStateTransformer (ProofStateTransformer), proofState )
+import qualified Kore.ProofState as ProofState.DoNotUse
```

Comment:
Well, our style guide recommends importing struct fields qualified with the struct name (the Naming section), so we can use them as, say, `StructName.field stuff` and have readable code.

However, most of the time we use these fields inside their structure, e.g. `StructName {field} = stuff`, so we don't need disambiguation. However, as the code we currently have shows, importing them unqualified will encourage people to use the field name as a getter without any qualifier.

The obvious solution would be to always import the field qualified with the struct name, but it gets tedious when importing multiple structures from the same file. So, the compromise that I found was to import the entire field with a qualifier that is obvious that should not be used. This allows using the field name inside their structure, but does not allow using them with the wrong qualifier, and makes the common case less tedious.

Does this seem reasonable?

Of course, in this exact case it may not make any difference, but it's easier to use the same pattern everywhere than to optimize locally. I can, of course, be less lazy and optimizer locally if you think it will make our code better by some metric (e.g. readability).

Reply:
I knew about importing field names qualified and then using them unqualified, but I never thought to do this, that's quite clever! Let's do it the way you have here, but I will think about if there is a better way to handle field selectors in general.